tests/dbfs/test_api.py

```diff
@@ -25,6 +25,8 @@
 from base64 import b64encode
 
 import os
+import time
```

Comment:
I think this is what `lint.sh` complains about. You probably forgot to remove it in your last commit.

Reply:
Thanks for pointing out.